src/API.jl

```diff
@@ -48,6 +48,10 @@ function Base.:(==)(a::PackageInfo, b::PackageInfo)
         a.source == b.source && a.dependencies == b.dependencies
 end
 
+function package_info(pkg::AbstractString)::PackageInfo
+    dependencies()[Base.identify_package(name).uuid]
```

Comment:
`identify_package` can return nothing, also I don't see a point in calling into full code loading when you have all the data in `dependencies()`.

Reply:
without `identify_package`, how can I get the UUID from the name?

Reply:
```julia
filter(p -> p.second.name == "Example", dependencies()) |> keys |> collect
```

If there is only one UUID that matches the desired name, you can do:

```julia
filter(p -> p.second.name == "Example", dependencies()) |> keys |> collect |> only
```